cstore_fdw.c

```diff
@@ -532,6 +531,45 @@ DroppedCStoreFilenameList(DropStmt *dropStatement)
 }
 
 
+/* FindCStoreTables returns list of CStore tables from given table list */
+static List *
+FindCStoreTables(List *tableList)
+{
+	List *cstoreTableList = NIL;
+	ListCell *relationCell = NULL;
+	foreach(relationCell, tableList)
+	{
+		RangeVar *rangeVar = (RangeVar *) lfirst(relationCell);
+
+		Oid relationId = RangeVarGetRelid(rangeVar, AccessShareLock, true);
+		if (CStoreTable(relationId))
+		{
+			cstoreTableList = lappend(cstoreTableList, rangeVar);
+		}
+	}
+
+	return cstoreTableList;
+}
+
+/* TruncateCStoreTable truncates given cstore tables */
+static void
+TruncateCStoreTables(List *cstoreTableList)
+{
+	ListCell *relationCell = NULL;
+	foreach(relationCell, cstoreTableList)
+	{
+		RangeVar *rangeVar = (RangeVar *) lfirst(relationCell);
+		Oid relationId = RangeVarGetRelid(rangeVar, AccessShareLock, true);
```

Comment:
One minor note, cstore does nothing on the following `truncate` [parameters](http://www.postgresql.org/docs/9.5/static/sql-truncate.html): `[ RESTART IDENTITY | CONTINUE IDENTITY ] [ CASCADE | RESTRICT ]`.
Do you think should we notice the users if those parameters exist (ie: if any parameter exists)?

I know `CASCADE` and `RESTRICT` cannot even be applicable since they rely on table constraints, where `cstore` does not allow at all.